README.md

```diff
@@ -1,18 +1,36 @@
-# wellness-check-bot
-This is a simple Rasa bot that uses a form to create a daily log of health information.
+# Wellness Check Bot
+[![Rasa](https://img.shields.io/badge/Rasa%20Open%20Source-2.4.3-purple.svg?style=flat-square)](https://rasa.com/docs/rasa/)
 
-## Running the assistant
-1. Install Rasa Open Source: https://rasa.com/docs/rasa/user-guide/installation/
-2. Train the model:
+This is a simple Rasa bot that uses a form to create a daily log of health information. User is asked several health related questions and at the end the logs are shared back to the user. This project demonstrates use of [Forms](https://rasa.com/docs/rasa/forms) & [FormValidationAction](https://rasa.com/docs/rasa/forms#advanced-usage).
 
-``rasa train``
+> **Note** : If you are looking for rasa version 1.x project files then head back to [master](https://github.com/kmanadkat/wellness-check-bot/tree/master) branch of this repository.
 
-3. Open a second terminal window and start the action server:
 
-``rasa run actions``
+## Installation and Setup
+It is recommended to use Python 3.6 or Python 3.7 with rasa framework. Below are the steps to setup virtual env, install rasa open source & running this project
 
-4. Return to the first terminal window and start the assistant on the command line:
+1. Clone current branch & open it's root directory in terminal.
 
-``rasa shell``
+2. Create virtual env with python: `python3.7 -m venv ./venv` and activate it by `source ./venv/bin/activate`
 
-![wellness-bot-conversation](https://github.com/karen-white/wellness-check-bot/blob/master/images/bot_conversation.png?raw=true)
+3. [Install rasa](https://rasa.com/docs/rasa/installation) `pip install rasa`
+
+### Generating Trained Model
+Make sure python virtual env is active in terminal & you are good to train model.
+```shell
+rasa train
+```
+
+### Running Actions Server
+After training is done, you need to run actions server to serve custom actions related computation.
+```shell
+rasa run actions
+```
+
+### Running Chatbot in Shell
+Below command loads the trained model, opens communication to custom actions running server & provides [REPL](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop) to interact with chatbot.
+```shell
+rasa shell
+```
+
+![wellness-bot-conversation](./images/bot_conversation_2.4.png)
```

Comment:
@karen-white preview on line 36 should work after code is merged because it works [here](https://github.com/kmanadkat/wellness-check-bot/tree/update-to-rasa-2.4/master)